scripts/libvirt_launcher.py

```
@@ -0,0 +1,754 @@
+    stopped = threading.Event()
+    receiver_error: list[BaseException] = []
+    log_file = open(log_path, "ab", buffering=0)
+
+    def receive() -> None:
+        try:
+            while not stopped.is_set():
+                chunk = stream.recv(65536)
+                if not chunk:
+                    break
+                _write_console_output(chunk, log_file)
+        except BaseException as exc:  # propagated after terminal restoration
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `BaseException`

(BLE001)